Gemfile

```diff
@@ -20,7 +20,7 @@ gem 'dotenv-rails', '~> 2.8'
 gem 'aws-sdk-s3', '~> 1.123', require: false
 gem 'fog-core', '<= 2.4.0'
 gem 'fog-openstack', '~> 1.0', require: false
-gem 'kt-paperclip', '~> 7.2'
+gem 'kt-paperclip', github: 'kreeti/kt-paperclip'
```

Comment:
Is it possible to wait until kt-paperclip has a release with this change?

Reply:
Yup. Opened as draft, will update when they do actual release.